plugin/evm/vm.go

```diff
@@ -141,15 +141,16 @@ const (
 	atomicTxGossipProtocol = 0x1
 
 	// gossip constants
-	txGossipBloomMaxItems          = 8 * 1024
```

Comment:
I think this change is worth a discussion (ok to leave as is)

Reply:
I'd prefer to leave it

Reply:
Updated

Reply:
would keeping this value same for new `OptimalParameters` mean that we expect 8*1024 entries in the filter?

Reply:
This actually does the same thing but the previous variable name was confusing (as you pointed out 😉).